integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -22,52 +22,49 @@ import java.sql.{DriverManager, ResultSet, Statement}
 import org.apache.spark.sql.SparkSession
```

Comment:
Please move this example to example folder, I suggest to put all example in one module, so instead of adding another example module for hive, I think it is better add it in existing example module and rename it to carbondata-example from carbondata-example-spark2

Reply:
@jackylk I have already moved this example to example folder and rename module named carbondata-example-spark2.